src/index.html

```diff
@@ -1,14 +1,44 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+
+      <div class="block_shell">
+
+        <div class="block">
+          <div class="block block_1">1</div>
+        </div>
+
+        <div class="block">
+          <div class="block block_2">2</div>
+        </div>
+
+        <div class="block">
+          <div class="block block_3">3</div>
+        </div>
+
+        <div class="block">
+          <div class="block block_4">4</div>
+        </div>
+
+        <div class="block">
+          <div class="block block_5">5</div>
+        </div>
+
+        <div class="block">
+          <div class="block block_6">6</div>
+        </div>
+
+      </div>
```

Comment:
```suggestion
      <div class="block_shell">
        <div class="block">1</div>
        <div class="block">2</div>
        <div class="block">3</div>
        <div class="block">4</div>
        <div class="block">5</div>
        <div class="block">6</div>
      </div>
```
Не понятно зачем тебе обертки вокруг этих блоков, должно хватить такой верстки